src/components/App.tsx

```diff
@@ -1,12 +1,46 @@
 import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import uuid from 'uuidv4';
+import './App.css';
+import TodoWrapper from './ToDoList/TodoWrapper';
+import { TaskType } from './ToDoList/types';
 
+const todoTasks: Array<TaskType> = [
```

Comment:
create mocks folder in the root and move tasks in new file

Reply:
done